mvp-spec/operator-design.md

```diff
@@ -52,197 +52,199 @@ Limitations:
 
 🟠 it can also appear in web server logs.
 
+
+## Write Scenario: the user visits a Publisher for the first time
+
+A user visits a Publisher and sets their preferences for the first time.
+
+In this case, the Publisher, the CMP and the Operator interact via 
+redirection mechanisms to collect the Pseudonymous-Identifier and the Preferences
+of the user and *write* it in a secure way. Therefore, this scenario is
+called the **"WRITE" scenario**.
+
 ```mermaid
 sequenceDiagram
-participant U as user
-participant B as browser
-participant P as publisher P http server
-participant C as CMP http server
-participant A as advertiser A http server
-participant O as operator node O
-
-rect rgba(224, 224, 224, .55)
-    note right of U: Time T1<br>"WRITE" scenario: user visits publisher<br>and sets their preferences for the first time
-
-    U ->> B: visit www.publisherP.com/pageP.html
-    activate U
-
-        activate B
-            B ->> P: GET /pageP.html
-        deactivate B
-
-        activate P
-            P -->> B: display
-        deactivate P
-
-        activate B
-            B ->> B: JS: no Prebid SSO ID 🍪 found
-            B -->> C: JS call: GET prebidURL?pubPage=publisherP.com/pageP.html
-        deactivate B
-
-        activate C
-            C ->> C: clear data = (timestamp<br>+ returnUrl=publisherP.com/pageP.html<br>+ sender=cmp.com)
-            C ->> C: SIGN (clear data + receiver=operatorO.prebidsso.com)<br/> with own private key
-            note over C: receiver is not part of the clear data
-            C -->> B: "read prebid SSO" URL = ...
-        deactivate C
-
-        activate B
-            B ->> B: JS: full page REDIRECT<br>(or wait for user click)
-            B ->> O: GET /readOrGetNewId?[...clear Data]&signature=xxx
-        deactivate B
-
-        activate O
-            note over O: sender == cmp.com<br>key == cmp.com public key
-            O ->> O: check cmp.com is allowed to read
-            O ->> O: VERIFY full data signature<br/>with cmp.com public key
-            note over O: cmp.com is authenticated<br/>and timestamp can be trusted
-            O ->> O: check timestamp is still valid
-            note over O: ok: this is a valid request
-            O ->> O: no 🍪 found
-            O ->> O: create ID
-            O ->> O: SIGN ID<br/>with own private key
-            note over O: ID has been signed by operatorO.prebidsso.com<br/>it can be trusted
-            O ->> O: clear data = (id + sign[id]<br>+ timestamp<br>+ sender=operatorO.prebidsso.com)
-            O ->> O: SIGN (clear data + receiver=publisherP.com)<br/> with own private key
-            O -->> B: REDIRECT
```

Comment:
I like these short sentences:
```
O ->> O: Create an ID and sign it because no 🍪 found
O ->> O: Craft a Publisher URL with ID, signature, timestamp and sign it
```
But for the rest, I think details are useful because this is the only place where they appear.

I would suggest this compromise:
```
activate O
            note over O: sender == cmp.com<br>key == cmp.com public key
            O ->> O: check cmp.com is allowed to read
            O ->> O: VERIFY full data signature<br/>with cmp.com public key
            note over O: cmp.com is authenticated<br/>and timestamp can be trusted
            O ->> O: check timestamp is still valid
            note over O: ok: this is a valid request
            O ->> O: Create an ID and sign it because no 🍪 found
            note over O: ID has been signed by operatorO.prebidsso.com<br/>it can be trusted
            O ->> O: clear data = (id + sign[id]<br>+ timestamp<br>+ sender=operatorO.prebidsso.com)
            O ->> O: Craft a Publisher URL with ID, signature, timestamp and sign it
            O -->> B: REDIRECT to the crafted URL
```

Reply:
What about providing details in text? The diagram would provide a high level of the flow to get the intents and the text would provide the details.